### DISCUSSION TITLE
Configurare sensore umidità Xiaomi Aqara WSDCGQ11LM su home assistant

### DISCUSSION THREAD

**Discussion by fregol** [indomus/forum#2899] [Hardware & Firmware]
Ciao ragazzi

premetto che sono davvero alle prime armi: ho installato HA (ultimo aggiornamento) su raspberry. 
Sto cercando di configurare un sensore xiaomi aqara WSDCGQ11LM (che è configurato sul suo hub aqara e visualizzabile su app xiaomi home per iphone) e ho seguito la guida ufficiale [(qui)](https://www.home-assistant.io/integrations/xiaomi_ble).
Mi viene richiesto un codice esadecimale AES a 32 caratteri. Ho usato il token extractor indicato nella guida ma ahimè questo codice per il sensore in questione non viene esposto (come invece avviene per altri igrometri della mi che possiedo):

![Immagine](https://github.com/indomus/forum/assets/48866573/2fc194f0-b950-480a-9cb4-37a328f6da34)

Sono rimasto bloccato e non so che fare! Qualche suggerimento/consiglio/aiuto!

Grazie mille in anticipo

**Comment by KC-inDomus** [comment]
Il WSDCGQ11LM è ZigBee, non puoi dunque integrarlo via Bluetooth, ma solo via ZibBee (con ZHA o ZigBee2MQTT)

**Reply by fregol** [reply]
azz! 
Grazie, dicevo io che c'era qualcosa che non andava...!
:D
